.github/workflows/ci-windows-installed.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     runs-on: windows-latest
     strategy:
       fail-fast: false
+      matrix:
+        compiler: [default, clang-cl]
```

Comment:
Another option is to use the `env` key:

https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

```
      env:
        CC: clang-cl
        CXX: clang-cl
```

Reply:
I wanted to do so except that I do not know what to put for MSVC. And setting env variables conditionally is roughly the same as what I did.